CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
+- Fix: Modify .test to upgrade to libmicrohttpd 0.9.75 (#4267)
```

Comment:
Looking to existing changelog, I think it would be better this:

```suggestion
- Hardening: upgrade microhttpd dependency from 0.9.70 to 0.9.76
```

(I'm using 0.9.76 above, assuming we are using that one at the end, send https://github.com/telefonicaid/fiware-orion/pull/4321#discussion_r1185078870)

Reply:
Updated in https://github.com/telefonicaid/fiware-orion/pull/4321/commits/bb03be6eb4e2cbfe7391bd693ac7ad1609a5cc64